buildres/linux/jabrefHost.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python -u
+
+# Note that running python with the `-u` flag is required on Windows,
+# in order to ensure that stdin and stdout are opened in binary, rather
+# than text, mode.
+
+import json
+import sys
+import struct
+import subprocess
+import shlex
+import os
+import logging
+from pathlib import Path
+
+# We assume that this python script is located in "jabref/lib" while the executable is "jabref/bin/JabRef"
+script_dir = os.path.dirname(os.path.realpath('__file__'))
+JABREF_PATH = os.path.join(script_dir, '../bin/JabRef')
+
+logging_dir = str(Path.home()) + "/.mozilla/native-messaging-hosts/"
+if not os.path.exists(logging_dir):
+    os.makedirs(logging_dir)
+logging.basicConfig(filename=logging_dir + 'jabref_browser_extension.log')
```

Comment:
Here I meant to say that the lines 17-23 are replaced by the ones in the comment
It's just a pathlib replacement, but github doesn't have a multiline suggestion yet

Reply:
If  you want I can make the changes directly to this branch..